misc-tools/dj_run_chroot.in

```diff
@@ -93,7 +109,7 @@ mount --bind /dev/pts "$CHROOTDIR/dev/pts"
 # Prevent perl locale warnings in the chroot:
 export LC_ALL=C
 
-[ -n "$INTERACTIVE" ] && echo "Entering chroot in '$CHROOTDIR'."
+[ -n "$INTERACTIVE" ] && echo "Entering chroot in '$CHROOTDIR'. Only [$CONCAT_SUBDIRMOUNTS] will be mounted during judgement. See chroot-start-stop.sh for the current list."
```

Comment:
This is now a very long line both in code and what it prints. Since POSIX `echo` doesn't support `-e` or newlines, I think this whole line should be included in the if statement below like
```
if [ -n "$INTERACTIVE" ]; then
  echo "Entering chroot in '$CHROOTDIR'."
  echo "Only [$CONCAT_SUBDIRMOUNTS] will be mounted during judgement. See chroot-start-stop.sh for the current list."
    chroot "$CHROOTDIR"
else
```

Reply:
I agree.